### PR TITLE
fix(module: modal): get rtl from ConfigProvider

### PR DESCRIPTION
**Pull request by zxyao145**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#2288 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  fix(module: modal): get rtl from ConfigProvider  |
| 🇨🇳 Chinese |   Modal：从ConfigProvider获取Rtl        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by hassan-gasemi — APPROVED** [review]
This commit solves the overlapping title and close button in Rtl mode. I've tested this change on local source.

File: components/modal/modalDialog/Modal.razor.cs [hunk 1]
```diff
@@ -213,8 +213,7 @@ private string GetWidth()
         /// <summary>
         /// Is RTL
         /// </summary>
-        [CascadingParameter]
-        public bool Rtl { get; set; } = false;
+        public bool Rtl => base.RTL;
```
Comment (ElderJames) [inline components/modal/modalDialog/Modal.razor.cs:216]: You can directly use `base.RTL`.

Comment (zxyao145) [reply components/modal/modalDialog/Modal.razor.cs:216]: > You can directly use `base.RTL`.

When using the template, it need to get RTL from the outside, but base.RTL is protected

Comment (ElderJames) [reply components/modal/modalDialog/Modal.razor.cs:216]: That's fine.

**Review by ElderJames — APPROVED** [review]

**Pull request merged**